src/components/social.js

```diff
@@ -17,20 +18,27 @@ module.exports = React.createClass({
             <div>{this.context.intl.formatMessage({id: 'help_spread_the_word'})}</div>
           </h3>
           <div className="row">
-            <div className="half" id="facebook">
+            <div className="third" id="facebook">
               <div className="social-button"></div>
               <a href={facebookShareURL} target="_blank">
-                <i className="fa fa-facebook fa-4x"></i>
+                <i className="fa fa-facebook fa-2x"></i>
                 <div>facebook</div>
               </a>
             </div>
-            <div className="half" id="twitter">
+            <div className="third" id="twitter">
               <div className="social-button"></div>
               <a href={twitterShareURL} target="_blank">
-                <i className="fa fa-twitter fa-5x"></i>
+                <i className="fa fa-twitter fa-2x"></i>
                 <div>twitter</div>
               </a>
             </div>
+            <div className="third" id="email">
+              <div className="social-button"></div>
+              <a href={emailShareURL} target="_blank">
+              <i className="fa fa-envelope fa-2x"></i>
+                <div>email</div>
```

Comment:
I guess the string is used for accessibility, so let’s localize it. It should be safe to reuse this string https://github.com/mozilla/donate.mozilla.org/blob/master/locales/en-US/messages.properties#L75
So something like this should work `<div>{this.context.intl.formatMessage({id: 'email'})}</div>`